tests/lib/MVC/Symfony/ErrorHandler/Php82HideDeprecationsErrorHandlerTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Tests\Core\MVC\Symfony\ErrorHandler;
+
+use Ibexa\Core\MVC\Symfony\ErrorHandler\Php82HideDeprecationsErrorHandler;
+use const PHP_VERSION_ID;
+use PHPUnit\Framework\TestCase;
+
+final class Php82HideDeprecationsErrorHandlerTest extends TestCase
+{
+    /** @var int */
+    private $originalErrorReporting;
```

Comment:
```suggestion
    private int $originalErrorReporting;
```
or it cannot be strictly typed for some reason?

Reply:
Unfortunately, yes: https://github.com/ezsystems/ezplatform-kernel/blob/1.3/composer.json#L10 `3.3` still works on PHP 7.3.